src/app/unlocked-stage/unlocked-stage.component.spec.ts

```diff
@@ -18,7 +21,7 @@ describe('UnlockedStageComponent', () => {
       declarations: [UnlockedStageComponent, ButtonNavComponent],
       providers: [LanguageService, HttpClient, HttpHandler, APIService,
         DashboardService, SharedDataService, ToastrService],
-      imports: [ToastrModule.forRoot(), RouterTestingModule]
+      imports: [ToastrModule.forRoot(), RouterTestingModule, MatSnackBarModule, MaterialModule]
```

Comment:
extra space after MatSnacBarModule,

Reply:
@shreyans29 I checked it. There is only one space on both the sides.